codecov.yml

```diff
@@ -8,8 +8,12 @@ coverage:
   range: "70...90"
 
   status:
-    project: yes
-    patch: yes
+    project:
+      default:
+        informational: true
```

Comment:
is the `informational` flag what prevents a decrease from causing a failure?

Reply:
i hope so :D

Reply:
That's what i'm wanting to test. It's supposed to stop any failure from happening. It will generate reports but not fail. But CI is down so i can't test.